### ISSUE TITLE
Typeahead Not Displaying Pre-filled Values

### ISSUE THREAD

**Issue by ryan-ethode** [bootstrap-tagsinput/bootstrap-tagsinput#359]
I'm using Boostrap-Tagsinput with TypeaheadJS in a Node/Sails project.

I have one page where tag values are selected in an input field, and are saved into a database. There is another page where users should be able to edit this selection.

Typically, I can populate the field value with an EJS variable, and the values that the user saved to the DB will get displayed in each field.

Here's the text input field in question:

```
<input type="text"  class="form-control getNameValue" name="infoType" autocomplete="off" data-role="tagsinput" value="<%=data.info_type%>">
```

After the page loads, this gets rendered out to:

```
<input type="text"  class="form-control getNameValue" name="infoType" autocomplete="off" data-role="tagsinput" value="data1, data1, data3">
```

On the frontend, the input field is blank however. The Typeahead tt-hint and tt-input fields are not getting populated with the values. The original tagsinput field _is_ getting these values correctly, but it is set to display:none because I am using Typeahead.

Does anyone know how to get pre-set values from my original field to display in my Typeahead field?

**Comment by Sumomannen** [comment]
Did you get this to work?

**Comment by ryan-ethode** [comment]
Sorta kinda. It worked, but it was after much cobbling together of mostly inefficient JS code. I'm not sure my code is even worth reposting. 

Basically I had my database pass me back an array of field values. I looped through those array values. I looped through the array, create a Typeahead tag div, and placed each value into a Typeahead tag div. While it looked okay on the frontend, there was a lot of duplication happening in the backend that any professional would definitely want to refactor.

Ultimately the desktop version of the app that I was working on was shelved in favor of the mobile app. It may come back to life later, but at the moment my code isn't in production anywhere. I could potentially PM my code to you if you'd like to see it.
